lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridge.cs

```diff
@@ -188,7 +191,13 @@ internal sealed class DriverBridge
             _driverRestartFailedEvaluatorHandlers = driverRestartFailedEvaluatorHandlers;
             _httpServerHandler = httpServerHandler;
 
-            _configurationProviders = new HashSet<IConfigurationProvider>(configurationProviders) { driverReconnConfigProvider };
+            var configurationProviderSet = new HashSet<IConfigurationProvider>(configurationProviders) { driverReconnConfigProvider };
+            var evaluatorConfig = TangFactory.GetTang().NewConfigurationBuilder().Build();
+            foreach (var configurationProvider in configurationProviderSet)
```

Comment:
Isn't there a way where you do something like `_configurationProviderString = serializer.ToString(Configurations.Merge( configurationProviderSet.Map(_ ->_.GetConfiguration())))`?

Reply:
Yes, I can do it in one line.